axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveCounterPartServiceImpl.java

```diff
@@ -16,45 +17,54 @@
 
   protected MoveRepository moveRepository;
   protected MoveLineToolService moveLineToolService;
+  protected MoveLineCreateService moveLineCreateService;
 
   @Inject
   public MoveCounterPartServiceImpl(
-      MoveRepository moveRepository, MoveLineToolService moveLineToolService) {
+      MoveRepository moveRepository,
+      MoveLineToolService moveLineToolService,
+      MoveLineCreateService moveLineCreateService) {
     this.moveRepository = moveRepository;
     this.moveLineToolService = moveLineToolService;
+    this.moveLineCreateService = moveLineCreateService;
   }
 
   @Override
   @Transactional
-  public void generateCounterpartMoveLine(Move move) {
+  public void generateCounterpartMoveLine(Move move) throws Exception {
```

Comment:
Need to manage rollback on the @Transactional annotation and throw the specific exceptions

Reply:
Need to use specific exception instead of Exception